src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue32041.cs

```diff
@@ -0,0 +1,47 @@
+#if ANDROID
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue32041 : _IssuesUITest
+{
+	public override string Issue => "Keyboard overlaps Entry when SoftInput.AdjustResize is set";
+
+	public Issue32041(TestDevice device) : base(device)
+	{
+	}
+
+	[Test]
+	[Category(UITestCategories.SafeAreaEdges)]
+	public void VerifyKeyboardDoesNotOverlapEntryWithAdjustResize()
```

Comment:
Add a test for AdjustPan Mode. Current test validates AdjustResize; adding a pan-mode assertion ensures no regressions where pan doesn't apply insets incorrectly.